static/images/how-it-works/how-it-works-1.svg

```diff
@@ -1 +1,46 @@
+<svg width="306" height="223" viewBox="0 0 306 223" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Not sure how much it matters, but should these be minified?
Happy to file a separate bug since its an uber-nit.